[CUBRIDMAN-182] support with deduplicate clause (#448)
CUBRID/cubrid-manual: en/admin/config.rst

**vacuum_ovfp_check_duration** specifies the duration for which data related to the count of index overflow pages, gathered by vacuum threads, is retained. Data that remains unchanged within the specified duration will be automatically removed. The unit of it's value is minutes.

**deduplicate_key_level**

>  Specifies the *deduplicate level* value implicitly specified in the index creation statement. Default is -1. See :doc:`/sql/schema/table_stmt`\ and :doc:`/sql/schema/index_stmt`.

Reviewer: ```suggestion
 specifies the *deduplicate level* value implicitly specified in the index creation statement. Default is -1. See :doc:`/sql/schema/table_stmt`\ and :doc:`/sql/schema/index_stmt`.
```

Author: @mhoh3963 
Shouldn't the first letter of a sentence be capitalized?

Reviewer 2: What about "It specifies ..."

Author: This part reflects the opinion of @hgryoo

>  Specifies the *deduplicate level* value implicitly specified in the index creation statement. Default is -1. See :doc:`/sql/schema/table_stmt`\ and :doc:`/sql/schema/index_stmt`.

Reviewer 3: The verb `specify` is repeated redundantly. And It seems to be better to add a reference to DEDUPLICATE.
 
```suggestion
 **deduplicate_key_level** determines the automatic inclusion and value setting of the WITH DEDUPLICATE statement within the index creation statement. For details on DEDUPLICATE, see :ref:`deduplicate_overview`. The default is -1(which means that the WITH DEDUPLICATE is not included implicitly).
```

 
.. note::

    *   If **deduplicate_key_level** is set to **-1**, even if the *deduplicate level* is explicitly specified in the CREATE INDEX statement, it is ignored and the *deduplicate level* is forced to 0.

**print_index_detail**

>  Specifies whether option information in the **WITH** clause is displayed when index syntax information is displayed, such as in the SHOW CREATE TABLE statement. Default is NO. However, the unloaddb tool is not affected by this setting.

Reviewer: ```suggestion
 specifies whether option information in the **WITH** clause is displayed when index syntax information is displayed, such as in the SHOW CREATE TABLE statement. Default is NO. However, the unloaddb tool is not affected by this setting.
```

Author: This part reflects the opinion of @hyunikn.
